packages/reactotron-server/src/app/main/main.tsx

```diff
@@ -1,6 +1,5 @@
 import React from "react"
 import ReactDOM from "react-dom"
-import { Text } from "reactotron-core-ui"
```

Comment:
I took this out, as I was getting compilation errors from Typescript with it in.

Reply:
To be clear, the error was simply that it was unused, but I couldnt build with it in.